### PR TITLE
Begin floorplan api

### PR DESCRIPTION
**Pull request by aviau**
- Create GET `/floorplans` endpoint
- Create GET`/floorplans/<id>/` endpoint
- Specify HTTP method in router.go
- Refactor handler functions. Instead of being a handler function, they are now functions that return a handler. We do it this way so that adding decorators is done in handlers.go and not in router.go. Router.go should only handle routes, and not decide whether a route is authenticated, returns json, or whatever else the decorators  do.

### CODE REVIEW

File: backend/api/handlers/lanets/router.go [hunk 1]
```diff
@@ -5,5 +5,5 @@ import (
 )
 
 func RegisterRoutes(r *mux.Router) {
-	r.Path("/").HandlerFunc(indexHandler)
+	r.Path("/").Handler(indexHandler()).Methods("GET")
```
Comment (aviau) [inline backend/api/handlers/lanets/router.go]: Only handle GETs

File: backend/api/handlers/lanets/handlers_test.go [hunk 2]
```diff
@@ -17,3 +17,13 @@ func TestIndexHandler(t *testing.T) {
 		t.Error("Expected Lan ETS logo ", res.Body.String())
 	}
 }
+
+func TextIndexHandlerPost(t *testing.T) {
```
Comment (aviau) [inline backend/api/handlers/lanets/handlers_test.go]: Now that we only handle gets, it should return a 404 on POST.

File: backend/api/handlers/lanets/handlers.go [hunk 3]
```diff
@@ -6,6 +6,9 @@ import (
 	"net/http"
 )
 
-func indexHandler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprint(w, lanetslogo.Logo)
+func indexHandler() http.Handler {
```
Comment (aviau) [inline backend/api/handlers/lanets/handlers.go]: Create a handler to respect the new standard

File: backend/api/handlers/floorplans/handlers.go [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+		if id != 1 {
+			http.NotFound(w, r)
+			return
+		}
+
+		// :)
+		fmt.Fprint(w, `{ "0": { name: "A-1", x: 957, y: 171, label: "A-1" } }`)
+	}
+
+	handler = decorators.JsonHeaders(handler)
```
Comment (aviau) [inline backend/api/handlers/floorplans/handlers.go]: This is an example of a decorator. The handler created above is decorated with JsonHeaders.

File: backend/api/handlers/lanets/handlers_test.go (hunk 2 above)
Comment (zergov) [inline backend/api/handlers/lanets/handlers_test.go]: Tested, it does.

File: backend/api/handlers/floorplans/handlers.go [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+func floorplansGetHandler() http.Handler {
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		// Bogus response for now :)
+		fmt.Fprint(w, "[]")
+	}
+
+	handler = decorators.JsonHeaders(handler)
```
Comment (zergov) [inline backend/api/handlers/floorplans/handlers.go]: swag :tada: 
header_is_json

File: backend/api/handlers/floorplans/handlers.go (hunk 4 above)
Comment (zergov) [inline backend/api/handlers/floorplans/handlers.go]: Bootiful

**Pull request merged**